examples/tests/cypress/support/commands.js

```diff
@@ -48,7 +48,15 @@ Cypress.Commands.add('sendEvent', (eventType, message) => {
 
 Cypress.Commands.add('assertReceivedEvents', (element, message) => {
   // assert number and content of received events
-  cy.get(`#${element}`)
+  cy.get(`#${element} > li`, {timeout: 30000})
+    .should('be.visible')
+    .first()
+    .contains(message)
```

Comment:
the name suggests that it checks a _list_ of _events_ against (a list of) already _received events_. What it _actually_ does is to check whether a given _message_ is _on top_ of the list of messages. Also, the element is not fixed, yet there are some assumptions: it must be a list, and - according to the function name or argument name - it must contain events or messages. Please refactor.

Reply:
Maybe something like `assertNewestEventContainsMessage` is better here?